hasura-cli/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:current-slim
+WORKDIR usr/src/app
+ENV NPM_CONFIG_PREFIX /home/node/.npm-global
+ENV PATH $PATH:/home/node/.npm-global/bin
+USER node
+RUN npm install -g hasura-cli
```

Comment:
looks like this is installing the latest (1.2.2) but the image is being tagged as 1.0.0, can you please confirm, and if possible, can you please pin the specific version to this `npm install` command